lib/process-services-cloud/src/lib/form/components/form-cloud.component.ts

```diff
@@ -96,19 +106,32 @@ export class FormCloudComponent extends FormBaseComponent implements OnChanges,
     @Output()
     formContentClicked = new EventEmitter<ContentLinkModel>();
 
+    /** Emitted when a display mode configuration is turned on. */
+    @Output()
+    displayModeOn = new EventEmitter<FormCloudDisplayModeConfiguration>();
+
+    /** Emitted when a display mode configuration is turned off. */
+    @Output()
+    displayModeOff = new EventEmitter<FormCloudDisplayModeConfiguration>();
+
     protected subscriptions: Subscription[] = [];
     nodeId: string;
     formCloudRepresentationJSON: any;
 
     protected onDestroy$ = new Subject<boolean>();
 
+    readonly id: string;
```

Comment:
can we use `formId` instead?

Reply:
As I said in a comment before, formId is not mandatory (the renderer can receive the form definition directly, instead of a taskId or a formId) and you can break situations where you render the same form twice in the same page.